### PR TITLE
UL-6797 add intent endpoint documentation

### PR DESCRIPTION
**Pull request by juan-caballero**
Intent documentation

### CODE REVIEW

File: source/includes/_intents.md [hunk 1]
```diff
@@ -41,22 +41,102 @@ Please do not share your access tokens with other users or applications and don'
...
+   "message":"this is a message", 
+   "conversationId":"12345678987654321"
+}
+```
+
+Field | Type  | Required | Description
+---------- | ---------- | ---------- | ------------------------------
+**botId** | string | true | ID of the bot that will sends the reply.<br />You can find the ID of your bot in our dashboard, or talk to your Customer Success Manger.
+**message** | string | true | The text message that the visitor typed into the chat.
+**conversationId** | string | false | An ID used by the chat provider to identify the conversation.<br />The chat provider can decide the format.
```
Comment (FranziAndFinn) [inline source/includes/_intents.md:60]: do we need a conversation id for the intents endpoint as well?

File: source/includes/_intents.md [hunk 2]
```diff
@@ -41,22 +41,102 @@ Please do not share your access tokens with other users or applications and don'
...
+
+Field | Type  | Description
+---------- | ---------- | ------------------------------
+**intents** | array[`Intent`] | Array containing intents that were predicted for the `message` sent in the request.<br /><br />Each object of type `Intent` contains the following fields:<br />**name** `string` - the intent's name<br />**confidence** `float` - Float value between 0 and 1 - giving the confidence with which this intent was predicted for the given `message`.
+**entities** | array [`Entity`] |  Array of entities that were recognized from the visitor message given in the request.<br /><br />Each object of type `Entity` will have the following fields:<br />**name** `string` - Name of the recognized entity<br />**confidence** `float` - Float value between 0 and 1 - giving the confidence with which this entity was recognized for the given `message`.<br />**form** `string` - Form of the entity recognized in the given `message`.<br />**index** `number` - TODO Index of the entity recognized.<br />**id** `number` - Identifier of the entity recognized.<br />**sanitize** `boolean` - Boolean marking if entity is sanitized.
+
+## Usage
+### Start a conversation
```
Comment (FranziAndFinn) [inline source/includes/_intents.md]: there's no starting of a conversation here, so i'd just stick to "usage" without a sub section

File: source/includes/_intents.md [hunk 3]
```diff
@@ -41,22 +41,102 @@ Please do not share your access tokens with other users or applications and don'
...
+curl 'https://chat.ultimate.ai/api/intents' \
+  -H 'Authorization: Basic {basicAuthToken}' \
+  -H 'content-type: application/json;charset=UTF-8' \
+  --data-raw '{"botId":"{botId}","conversationId":"{conversationId}","message":"{message}"}' \
+  --compressed
+```
+
+```javascript
+const getIntents = fetch(
+        'https://chat.ultimate.ai/api/v2/automation',
```
Comment (FranziAndFinn) [inline source/includes/_intents.md]: `/api/intents`

File: source/includes/_intents.md [hunk 4]
```diff
@@ -41,22 +41,102 @@ Please do not share your access tokens with other users or applications and don'
```
Comment (FranziAndFinn) [inline source/includes/_intents.md:41]: could you add a short into to what we'd use this endpoint for.
check here for inspiration: https://ultimateai.github.io/public-api/#chat-automation
basically something along the lined of "get the intent for a given body of text, extract entities etc."

File: source/includes/_intents.md (hunk 1 above)
Comment (juan-caballero) [reply source/includes/_intents.md:60]: It is an optional parameter, that is only used to create the session.

File: source/includes/_intents.md (hunk 2 above)
Comment (juan-caballero) [reply source/includes/_intents.md]: copy-paste god, forgive me!

File: source/includes/_intents.md (hunk 3 above)
Comment (juan-caballero) [reply source/includes/_intents.md]: copy-paste again

File: source/includes/_intents.md (hunk 4 above)
Comment (juan-caballero) [reply source/includes/_intents.md:41]: Not sure what you mean here. Is it not what I put at the top of the file? => 
`Send a visitor message from a chat or any type of text to get a list of matching intents with their confidences as well as a list of entities we extracted from the text. Use this endpoint to classify or label any piece of text.`

Comment (FranziAndFinn) [reply source/includes/_intents.md:41]: you're right, didn't see that

**Review by FranziAndFinn — APPROVED** [review]

**Comment by FranziAndFinn** [comment]
sorry, i'm the slowest reviewer in the world, approved now ;)

**Pull request merged**